package.json

```diff
@@ -34,6 +34,7 @@
     "eslint-plugin-testing-library": "^5.0.0"
   },
   "dependencies": {
+    "moment": "^2.29.1",
```

Comment:
Moment.js has a few flaws such as data mutability and bundle size. We can keep it for now but consider some alternatives in the future. This is not blocking the MVP.

<img width="1324" alt="Screenshot 2021-12-12 at 10 02 54" src="https://user-images.githubusercontent.com/608862/145708129-29222fd1-32d2-439a-9c5d-ccf84ea15e09.png">